deploy/olm-catalog/kubeflow/1.2.0/kubeflow.v1.2.0.clusterserviceversion.yaml

```diff
@@ -13,11 +13,20 @@ metadata:
         },
         "spec": {
             "applications": [
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                      "name": "manifests",
+                      "path": "openshift/openshift-scc/base"
```

Comment:
scc is for openshift only. If you are pushing this to the community operator, their CI is deploying it on regular kubernetes.

For the openshift version of this operator, the Red Hat team is maintaining it and they call it OpenDataHub.